packages/fxa-payments-server/src/components/SubscriptionTitle/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+import './index.scss';
+
+type SubscriptionTitleProps = {
+  screenType: 'create' | 'success' | 'processing';
+  className?: string;
+};
+
+export const SubscriptionTitle = ({
+  screenType,
+  className = '',
+}: SubscriptionTitleProps) => {
+  const defaultHeaders: { [index: string]: string } = {
```

Comment:
The type here can be `{ K in SubscriptionTitleProps['screenType']: string }` if the three keys are meant to match the three values for `screenType`.

Reply:
This throws a `prettier` error on the git commit hook...

Reply:
Oh sorry, yeah missed some brackets there.